storey/drivers.py

```diff
@@ -126,6 +130,19 @@ async def _load_schema(self, container, table_path):
 
     async def _save_key(self, container, table_path, key, aggr_item, partitioned_by_key, additional_data=None):
         self._lazy_init()
+
+        # test whether server support parallel operations
+        if self._parallel_ops:
+            try:
+                test_expression = "a=init_array(2,'double',0.0);a[0..1]=pmax(a[0..1], a[0..1]);delete(a);"
+                response = await self._v3io_client.kv.update(container, table_path, str(key),
+                                                             expression=test_expression, condition="",
+                                                             raise_for_status=v3io.aio.dataplane.RaiseForStatus.never)
+                if response.status_code != 200:
+                    self._parallel_ops = False
```

Comment:
Might wanna log a warning there.

Reply:
+1
log what type of operation is going to be used + error code in debug level